xcode/Subconscious/Shared/Components/Common/Buttons/GhostPillButtonStyle.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  GhostPillButtonStyle.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 30/3/2023.
+//
+
+import SwiftUI
+
+struct GhostPillButtonStyle: ButtonStyle {
```

Comment:
Introduced to match the twitter follow vs following distinction:

<img width="390" alt="image" src="https://user-images.githubusercontent.com/5009316/229039145-51a4144c-021e-4bf5-9d63-fe5916f12903.png">